python/lsst/pipe/tasks/visualizeVisit.py

```diff
@@ -0,0 +1,250 @@
+# This file is part of pipe_tasks.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+import numpy as np
+
+import lsst.pex.config as pexConfig
+import lsst.pipe.base as pipeBase
+import lsst.pipe.base.connectionTypes as cT
+import lsst.afw.math as afwMath
+import lsst.afw.image as afwImage
+import lsst.afw.cameraGeom.utils as afwUtils
+
+
+__all__ = ['VisBinConfig', 'VisBinTask', 'VisMosaicConfig', 'VisMosaicTask']
+
+
+class VisBinConnections(pipeBase.PipelineTaskConnections,
+                        dimensions=("instrument", "detector")):
+    inputExp = cT.Input(
+        name="calexp",
+        doc="Input exposure data to mosaic.",
+        storageClass="ExposureF",
+        dimensions=("instrument", "detector"),
+    )
+    camera = cT.PrerequisiteInput(
+        name="camera",
+        doc="Input camera to use for mosaic geometry.",
+        storageClass="Camera",
+        dimensions=("instrument", "calibration_label"),
+    )
+
+    outputExp = cT.Output(
+        name="calexpBin",
+        doc="Output binned image.",
+        storageClass="ExposureF",
+        dimensions=("instrument", "detector"),
+    )
+
+
+class VisBinConfig(pipeBase.PipelineTaskConfig,
+                   pipelineConnections=VisBinConnections):
+    """Configuration for focal plane visualization.
+    """
+    binning = pexConfig.Field(
+        dtype=int,
+        default=8,
+        doc="Binning factor to apply.",
+    )
+    detectorKeyword = pexConfig.Field(
+        dtype=str,
+        default='DET-ID',
+        doc="Metadata keyword to use to find detector if not available from input.",
+    )
+
+
+class VisBinTask(pipeBase.PipelineTask,
```

Comment:
It is more verbose, but I think our coding standards perfer longer descriptive names, so something like VisualizeBinTask. On that note, I am not sure what the "bin" is in this name, maybe be more descriptive there.

Reply:
Makes sense to me.  Renamed to Visualize{Action}ExpTask, as they are tasks that Bin/Mosaic Exposure data for Visualization.